_posts/2018-12-17-pool-conversion-tool

```diff
@@ -0,0 +1,144 @@
+---
+title: Pool conversion tool
+author: Plewa Łukasz
+layout: post
+---
+
+## Introduction
+
+When we published the first PMDK stable release, we committed to maintaining
+the stable on-media layout of persistent memory pools, so all further PMDK
+changes had to be backward compatible. But we failed to accomplish this goal,
+so we introduced a ``pmempool convert`` command. This tool was used to convert
+pools created with previous PMDK versions, to the newer pool layout.
```

Comment:
This tool was used to convert pools which were created with old PMDK versions to the newer on-media layout.